cleverhans/loss.py

```diff
@@ -329,3 +334,100 @@ def fprop(self, x, y, **kwargs):
                   "MixUp. LossFeaturePairing may be removed "
                   "on or after 2019-03-06.")
     return loss
+
+
+class SNNLCrossEntropy(CrossEntropy):
+  EPSILON = 0.00001
```

Comment:
Is it possible to use a different name, or a more descriptive name? Throughout most of the library we use `eps` to mean the size of the norm ball for a threat model.